packages/fxa-graphql-api/src/lib/server.ts

```diff
@@ -31,25 +32,26 @@ export async function createServer(
 ): Promise<ApolloServer> {
   const schema = await TypeGraphQL.buildSchema({
     container: Container,
-    resolvers: [AccountResolver],
+    resolvers: [AccountResolver]
   });
+  const authUser = Container.get(SessionTokenAuth);
   const debugMode = config.env !== 'production';
   const defaultContext = async ({ req }: any) => {
     const bearerToken = req.headers[config.authHeader.toLowerCase()];
-    const userId = bearerToken.split('Bearer ')[1];
+    const userId = await authUser.lookupUserId(bearerToken);
```

Comment:
What happens if `lookupUserId` throws? Should this line be wrapped in a try/catch, and rethrown as a more generic error?

Reply:
If it throws, then we'll get a sentry error reported. There's another graphql hook we can use to rewrite the top level error that would be a good spot to do that. But I'm not yet what those should be, which will probably be hard to determine until we start running it.